src/plugin/featureaction/featurestyleaction.js

```diff
@@ -306,7 +306,8 @@ plugin.im.action.feature.StyleAction.prototype.fromXml = function(xml) {
         fillColor = os.style.toRgbaString(fillColorArr);
       }
 
-      os.style.setConfigColor(styleConfig, fillColor, [os.style.StyleField.FILL]);
+      // Only change the fill color without changing the image fill color too
+      styleConfig.fill.color = fillColor;
```

Comment:
The config properties should use bracket notation (`styleConfig['fill']['color']`), to avoid being renamed by the compiler which would break this functionality in the compiled app.